### PR TITLE
[Site] [Dark Mode] Update import catalog modal according to theme #1203

### LINKED ISSUE #1203: [Site] [Dark Mode] Update import catalog modal according to theme
#### Current State of the Meshery.io site
Dark mode issues with modals:
The text is not visible in dark mode. 
1. Import catalog item modal.
image

#### Desired State of the Meshery.io site
Update modal text and icons according to dark mode. 

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by vaibhav-xt**
**Description**
There is no option left except this. If we declare a sass variable, it conflicts with the dark and light theme colors. @sudhanshutech has tried before this is pr #1114 and it was closed. 
This PR fixes #1203 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
hmm build is failing..
@vaibhav-xt Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. :)

**Comment by samyakshah3008** [comment]
@vaibhav-xt  Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would. :)

**Comment by vaibhav-xt** [comment]
Hello @samyakshah3008 & @Chadha93, please get a review and I think it's ready to merge.

File: _sass/catalog.scss [hunk 1]
```diff
@@ -9,6 +9,9 @@
   font-size: 18px;
   font-weight: 400;
 }
+.tabcontent h3{
+  color: #000;
```
Comment (Chadha93) [inline _sass/catalog.scss:13]: use theme ref

File: _sass/catalog.scss [hunk 2]
```diff
@@ -276,6 +279,11 @@ p {
   text-align: center;
   padding: 15px 0 15px 0;
 }
+
+.pattern-name-comingsoon{
+  color: #fff;
```
Comment (Chadha93) [inline _sass/catalog.scss:284]: here too

**Review by Chadha93 — CHANGES_REQUESTED** [review]

**Comment by Chadha93** [comment]
@vaibhav-xt Review changes pending...

**Comment by thisiskaransgit** [comment]
Hey @vaibhav-xt, thank you for your efforts here. The implemented change does not go well with the website.
- The change of text color is desirable, what we need to change is the modal color according to the theme, the same way we implemented it in the catalog modal. This change will be easily done by referencing the color variable.
 ````scss
   background-color: var(--brand-color-secondary);
 ````

- For the coming soon card, 
 1. Remove the declaration of h5 style from `programs.scss` and add it to the element style file
 ```scss
h4, h5, h6{
	color: var(--color-secondary-dark);
}
``` 
 1. Make the style consistent, by removing the background shade.
 
 The issue that you described was due to the presidency of styles, by following the above steps it will be fixed for this section.

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
Hey @vaibhav-xt, thank you for your efforts here. The implemented change does not go well with the website.
- The change of text color is desirable, what we need to change is the modal color according to the theme, the same way we implemented it in the catalog modal. This change will be easily done by referencing the color variable.
 ````scss
   background-color: var(--integrations-bg);
 ````
  Also, change the tab heading color to Kepple green
- For the coming soon card, 
 1. Remove the declaration of h5 style from `programs.scss` and add it to the element style file
 ```scss
h4, h5, h6{
	color: var(--color-secondary-dark);
}
``` 
 2. Make the style consistent, by removing the background shade.
 3. Change the overlay-text to be dark mode compatible
 
 The issue that you described was due to the presidency of styles, by following the above steps it will be fixed for this section.

**Review by Chadha93 — COMMENTED** [review]
@vaibhav-xt Please also add links to meshery docs for UI and mesherctl in the modal section.

**Review by thisiskaransgit — COMMENTED** [review]
Hey @vaibhav-xt, Rest of the changes looks good, but the pattern-id is not visible 

 in _sass/catalog.scss
 Remove color style
 ```scss
pattern-id-overlay{
color: white
}
``` 
Do this and we're done here

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

File: _includes/help-modal.html [hunk 3]
```diff
@@ -98,7 +101,8 @@ <h3>Meshery UI</h3>
           </div>
           
           <div id="MesheryCLI" class="tabcontent" style="display: none;">
-            <h3>Meshery CLI</h3>
+            <h3><a href="https://docs.meshery.io/project/contributing/contributing-cli">Meshery CLI</a></h3>
```
Comment (thisiskaransgit) [inline _includes/help-modal.html:104]:
```suggestion
            <h3>Meshery CLI</h3>
            <p>To use mesheryctl follow the steps mentioned in the <a href="https://docs.meshery.io/installation/mesheryctl">docs</a></p>
            
```

File: _includes/help-modal.html [hunk 4]
```diff
@@ -75,14 +77,15 @@
...
+            <span class="closed" style=" padding: 4px 10px">&times;</span>
             <button class="tablinks active" onclick="openTab(event, 'MesheryUI')">Meshery UI</button>
             <button class="tablinks" onclick="openTab(event, 'MesheryCLI')">Meshery CLI</button>
           </div>
           
           <!-- Tab content -->
           <div id="MesheryUI" class="tabcontent">
-            <h3>Meshery UI</h3>
+            <!-- <h3>Meshery UI</h3> -->
+            <h3><a href="https://docs.meshery.io/project/contributing/contributing-ui">Meshery UI</a></h3>
```
Comment (thisiskaransgit) [inline _includes/help-modal.html:88]:
```suggestion
            <h3>Meshery UI</h3>
            <p>To install Meshery follow the steps mentioned in the  <a href="https://docs.meshery.io/installation/quick-start#install-meshery">docs</a></p>
            
```

**Review by thisiskaransgit — COMMENTED** [review]
@Chadha93

File: _includes/help-modal.html [hunk 5]
```diff
@@ -75,14 +77,16 @@
...
-            <span class="closed" style=" padding: 4px 4px">&times;</span>
+            <span class="closed" style=" padding: 4px 10px">&times;</span>
             <button class="tablinks active" onclick="openTab(event, 'MesheryUI')">Meshery UI</button>
             <button class="tablinks" onclick="openTab(event, 'MesheryCLI')">Meshery CLI</button>
           </div>
           
           <!-- Tab content -->
           <div id="MesheryUI" class="tabcontent">
             <h3>Meshery UI</h3>
+            <p>To install Meshery follow the steps mentioned in the  <a href="https://docs.meshery.io/installation/quick-start#install-meshery">docs</a></p>
```
Comment (Chadha93) [inline _includes/help-modal.html:88]:
```suggestion
            <p>To import designs, visit <a href="https://docs.meshery.io/guides/meshery-design#import-your-designs">docs</a></p>
```

File: _includes/help-modal.html [hunk 6]
```diff
@@ -99,6 +103,9 @@ <h3>Meshery UI</h3>
           
           <div id="MesheryCLI" class="tabcontent" style="display: none;">
             <h3>Meshery CLI</h3>
+            <p>To use mesheryctl follow the steps mentioned in the <a href="https://docs.meshery.io/installation/mesheryctl">docs</a></p>
```
Comment (Chadha93) [inline _includes/help-modal.html:106]:
```suggestion
            <p>Import using mesheryctl, visit <a href="https://docs.meshery.io/installation/mesheryctl">docs</a> for steps.</p>
```

**Pull request merged**